third_party/plugin/src/main/BUILD

```diff
@@ -1,6 +1,6 @@
 licenses(["notice"])  # 3-clause BSD
 
-load("//scala:scala.bzl", "scala_library", "scala_library_for_plugin_bootstrapping")
```

Comment:
Not a big deal, but I don't understand why this change is necessary.

Reply:
Nothing related to this PR, I think I played around with the file when getting tests to pass and noticed it wasn't used